JULIERAJ/KIDS-FIRST-Front
# Fix/121/sidebar header active state

## One Line Description
Fix the sidebar header active state

## Test Steps
1. Open the application.
2. Navigate between the Home and Calendar pages and verify that the header and sidebar have the same name, and the 
    corresponding item in the sidebar is focused.
3. Ensure that all buttons are fully clickable.

## Checklist
